tools/arxiv/monitoring/weekend_monitor.py

```
@@ -0,0 +1,356 @@
+"""
+
+import os
+import sys
+import json
+import time
+import psutil
+import sqlite3
+import subprocess
+from pathlib import Path
+from datetime import datetime, timedelta
+from typing import Dict, List, Optional, Tuple
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)